Anna_Gamzinova/HW_18_framework_from_scratch/page_objects/finish_page.py

```diff
@@ -0,0 +1,17 @@
+from Anna_Gamzinova.HW_18_framework_from_scratch.utilities.web_ui.base_page import BasePage
+from Anna_Gamzinova.HW_18_framework_from_scratch.utilities import locators
+
+
+class FinishPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    def is_thanks_header(self):
+        return self.is_displayed(locators.thanks_header)
+
+    def label_picture(self):
```

Comment:
if function return bool it's name should start with is

Reply:
fixed